dev/cognitiveservices/data-plane/Language/examples/text/SuccessfulLanguageDetectionRequest.json

```diff
@@ -37,8 +37,7 @@
               "detectedLanguage": {
                 "confidenceScore": 1,
                 "iso6391Name": "en",
-                "name": "English",
-                "script": "Unknown"
+                "name": "English"
```

Comment:
shouldn't we keep it here and set it to be Latin?

Reply:
The model can currently detect Latin script for Indian languages like Hindi etc. These languages can be written in native script as well as in Latin ( both of which are detected by the model ) and is a use case for our customers. It doesn't detect script for English and so this field was removed.

Reply:
@deyaaeldeen We made the "Script" parameter as optional as per suggestion by review board. the "Script" parameter is populated for particular model version & for queries which are in that particular locale. For the example here, that parameter will not be outputted (will be set to null)

Reply:
That makes sense, thanks for clarifying. The PR looks good but it seems like there is no example that demonstrates `script`. Any chance one can be added?

Reply:
@deyaaeldeen added an example with script.

Reply:
@deyaaeldeen please merge the PR if it looks good. Says I am not authorized to merge it. Thanks

Reply:
I pinged @lmazuel to merge it.